levels/checkout_file_from_branch.rb

```diff
@@ -0,0 +1,35 @@
+difficulty 2
+description "Grab the fixed app.rb file from the feature branch into master, but don't get the README.md file from that branch. Try to do it without copying it manually!"
+
+setup do
+  repo.init
+  FileUtils.touch("app.rb")
+  FileUtils.touch("README.md")
+  repo.add("app.rb")
+  `echo "Some buggy code" >> app.rb`
```

Comment:
This is a great level idea.

Would it be possible to change this to use `File.write` instead of `echo` here?

```
repo.init
File.write("app.rb", "Some buggy code")
repo.add("app.rb")
...
```

Reply:
Thanks, I'm glad you like it! I just swapped out `echo` for `File.write`, and it worked the same as previously when I tested it locally.